Dockerfile

```diff
@@ -0,0 +1,52 @@
+################
+##### Builder
+##### docker buildx create --use --name multi-builder --platform linux/arm64,linux/amd64
+# https://github.com/docker/buildx/issues/318#issuecomment-1023226339
+#FROM --platform=$BUILDPLATFORM rustlang/rust:nightly-buster-slim as builder
+FROM rustlang/rust:nightly-buster-slim as builder
```

Comment:
I think we should define the base image with `ARG` , using stable as the default toolchain. The reason for customizing base image is to stay in sync with the CI, which builds on stable, beta, nightly and MSRV toolchains: https://github.com/rust-lang/mdBook/blob/master/.github/workflows/main.yml